app/assets/javascripts/Results/keybinding.js

```diff
@@ -22,6 +22,14 @@ Mousetrap.bind("shift+right", function () {
   }
 });
 
+// Go to random submission with r
```

Comment:
It's great that you thought about adding a keyboard shortcut. In terms of the actual shortcut to add, it would be better to make it similar to the one for "next grouping". Something like `ctrl+shift+right`. Also make sure to update this comment.

Also, make sure this keyboard shortcut appears on the help modal (accessible when you press `?`)